spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe 'sign in' do
+  let(:user) do
+    Fabricate(:user, email: 'user@timeoverflow.org', password: 'papapa22')
+  end
+
+  context 'with a valid password' do
+    it 'signs the user in' do
+      sign_in_with(user.email, 'papapa22')
+      expect(page).to have_content(I18n.t('application.navbar.sign_out'))
```

Comment:
it checks that the log out link is present. That means we succesfully logged in.